<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSensorDialog</class>
 <widget class="QDialog" name="NewSensorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lattitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Longtitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mac</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Func 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Func 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Func 3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMacNew">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLatNew">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLngNew">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxFunc1New">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Enable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Disable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not Support</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbxFunc2New">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Enble</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Disable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not Support</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbxFunc3New">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Enable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Disable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not Support</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineClusterNew">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cluster head</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewSensorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewSensorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
